modules/IndexRedirect.js

```diff
@@ -41,4 +21,24 @@ class IndexRedirect extends Component {
 
 }
 
+IndexRedirect.propTypes = {
+  to: string.isRequired,
+  query: object,
+  state: object,
+  onEnter: falsy,
+  children: falsy
+}
+IndexRedirect.createRouteFromReactElement = function (element, parentRoute) {
```

Comment:
Yes, that and the two spaces after the `createRouteFromReactElement function`. It looks like that's just shifted up one line.